x-pack/test/detection_engine_api_integration/security_and_spaces/tests/telemetry/detection_rules.ts

```diff
@@ -418,7 +1235,227 @@ export default ({ getService }: FtrProviderContext) => {
+            getInitialDetectionMetrics().detection_rules.detection_rule_usage.custom_total
+          );
+        });
+      });
+
+      it('should show "legacy_notifications_disabled" to be "1" for rule that has at least "1" action(s) and the alert is "disabled"/"in-active"', async () => {
+        await installPrePackagedRules(supertest, log);
+        // Rule id of "9a1a2dae-0b5f-4c3d-8305-a268d404c306" is from the file:
+        // x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/elastic_endpoint_security.json
+        const immutableRule = await getRule(supertest, log, '9a1a2dae-0b5f-4c3d-8305-a268d404c306');
+        const hookAction = await createNewAction(supertest, log);
+        const newRuleToUpdate = getSimpleRule(immutableRule.rule_id, false);
+        await updateRule(supertest, log, newRuleToUpdate);
+        await createLegacyRuleAction(supertest, immutableRule.id, hookAction.id);
+
+        await retry.try(async () => {
+          const stats = await getStats(supertest, log);
+          // We have to search by "rule_name" since the "rule_id" it is storing is the Saved Object ID and not the rule_id
+          const foundRule = stats.detection_rules.detection_rule_detail.find(
+            (rule) => rule.rule_id === '9a1a2dae-0b5f-4c3d-8305-a268d404c306'
+          );
+          if (foundRule == null) {
+            throw new Error('Found rule should not be null. Please change this end to end test.');
+          }
+          const {
+            created_on: createdOn,
+            updated_on: updatedOn,
+            rule_id: ruleId,
+            ...omittedFields
+          } = foundRule;
+          expect(omittedFields).to.eql({
+            rule_name: 'Simple Rule Query',
+            rule_type: 'query',
+            rule_version: 3,
+            enabled: false,
+            elastic_rule: true,
+            alert_count_daily: 0,
+            cases_count_total: 0,
+            has_notification: false,
+            has_legacy_notification: true,
+          });
+          expect(
+            stats.detection_rules.detection_rule_usage.elastic_total.notifications_disabled
+          ).to.eql(0);
+          expect(
+            stats.detection_rules.detection_rule_usage.elastic_total.legacy_notifications_enabled
+          ).to.eql(0);
+          expect(
+            stats.detection_rules.detection_rule_usage.elastic_total.legacy_notifications_disabled
+          ).to.eql(1);
+          expect(
+            stats.detection_rules.detection_rule_usage.elastic_total.notifications_enabled
+          ).to.eql(0);
+          expect(stats.detection_rules.detection_rule_usage.custom_total).to.eql(
+            getInitialDetectionMetrics().detection_rules.detection_rule_usage.custom_total
+          );
+        });
+      });
+
+      it('should show "legacy_notifications_enabled" to be "1" for rule that has at least "1" action(s) and the alert is "enabled"/"active"', async () => {
```

Comment:
Do you want to just call out that `has_legacy_notification` should be `true` here as well to let future test readers to look out for it?